subiquity/common/types.py

```diff
@@ -290,6 +290,15 @@ class Partition:
     path: Optional[str] = None
 
 
+@attr.s(auto_attribs=True)
+class ZPool:
+    pool: str
+    mountpoint: str
+    zfses: Optional[str] = None  # wrong but I'm not doing the ZFS objects yet
```

Comment:
Er well ok. Optional[str] definitely seems like the wrong annotation though? It's a list of ... well... somethings?

Reply:
Done